common/lib/xmodule/xmodule/modulestore/split_mongo/mongo_connection.py

```diff
@@ -579,13 +581,7 @@ def close_connections(self):
         """
         Closes any open connections to the underlying databases
         """
-        self.database.connection.close()
-
-    def mongo_wire_version(self):
-        """
-        Returns the wire version for mongo. Only used to unit tests which instrument the connection.
-        """
-        return self.database.connection.max_wire_version
+        self.database.client.close()
```

Comment:
@giovannicimolin, same as the comment above.

Reply:
@lgp171188 See my reply on the other comment.